std/machines/hash/keccakf16_memory.asm

```diff
@@ -0,0 +1,876 @@
+use std::array;
+use std::utils;
+use std::utils::unchanged_until;
+use std::utils::force_bool;
+use std::convert::expr;
+use std::convert::int;
+use std::convert::fe;
+use std::prelude::set_hint;
+use std::prelude::Query;
+use std::prover::eval;
+use std::prover::provide_value;
+use std::machines::small_field::memory::Memory;
+use std::machines::small_field::add_sub::AddSub;
+
+machine Keccakf16Memory(mem: Memory, add_sub: AddSub) with
+    latch: first_step,
+    operation_id: operation_id,
+    call_selectors: sel,
+{
+    // ------------- Begin memory read / write ---------------
+    // Additional columns compared to the non-memory version:
```

Comment:
You can use `/* ... */` comments here so you don't have to start every line with `//`

Reply:
Smart. Done.